packages/workflow/src/interfaces.ts

```diff
@@ -1147,6 +1147,10 @@ export interface INode {
 	webhookId?: string;
 	extendsCredential?: string;
 	rewireOutputLogTo?: NodeConnectionType;
+	forceCustomOperation?: {
```

Comment:
Can we add an explanatory docline? This might not be self-evident to someone unfamiliar.